### ISSUE TITLE
Breaking Change with ansible.netcomm v6.1.2

### ISSUE THREAD

**Issue by happly** [ansible-collections/cisco.ios#1079]
##### SUMMARY

Please refer to https://github.com/ansible-collections/ansible.netcommon/issues/639 for the full details of the issue.

ansible.netcommon v6.1.2 which gets pulled in as a dependency causes at least cisco.ios.ios_interfaces to break.

pinning ansible.netcommon to v6.1.1 avoids the breaking change.

I'm not quite sure whether the fix is an upstream/downstream one, so I created a Bug Report here too as well as to document a work around for anyone experiencing the same issue.

##### ISSUE TYPE
- Bug Report

##### ANSIBLE VERSION

```paste below

```

##### COLLECTION VERSION

```paste below

```

##### CONFIGURATION

```paste below

```

##### STEPS TO REPRODUCE

```yaml

```

##### ACTUAL RESULTS

```paste below
{
  "module_stdout": "",
  "module_stderr": "edit_config() got an unexpected keyword argument 'err_responses'",
  "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error",
  "_ansible_no_log": false,
  "changed": false
}
```

**Comment by NilashishC** [comment]
@happly This has been fixed with https://github.com/ansible-collections/ansible.netcommon/pull/640. We will do a netcommon release soon.

**Comment by NilashishC** [comment]
@happly @jorgenspange we have just released v6.1.3 of ansible.netcommon, could you please update? This should fix this issue for you. Thanks!

**Comment by happly** [comment]
Confirming that 6.1.3 resolves the issue.
